presto-tests/src/main/java/com/facebook/presto/tests/AbstractTestQueries.java

```diff
@@ -679,7 +679,7 @@ public void testValues()
         assertQuery("SELECT * FROM (VALUES (1.1, 2), (sin(3.3), 2+2))");
         assertQuery(
                 "SELECT * FROM (VALUES (1.1, 2), (sin(3.3), 2+2)) x (a, b) LEFT JOIN (VALUES (1.1, 2), (1.1, 2+2)) y (a, b) USING (a)",
-                "VALUES (1.1, 2, 1.1, 4), (1.1, 2, 1.1, 2), (sin(3.3), 4, NULL, NULL)");
+                "VALUES (1.1, 2, 4), (1.1, 2, 2), (sin(3.3), 4, NULL)");
```

Comment:
If these queries are covered by `TestJoinUsing`, then maybe we could remove them from here?